hammerdb/sql/ch-benchmark-distribute.sql

```diff
@@ -0,0 +1,3 @@
+SELECT create_reference_table('region');
```

Comment:
As far as I know the data size for ch benchmark tables are independent of the scale factor. These are small tables, H part uses both these tables and the tpcc tables. So H part scales with scale factor because of the TPCC part, correct me if I am wrong @marcocitus.